pkg/api/volume_req.go

```diff
@@ -137,6 +137,20 @@ type VolEditReq struct {
 	DeleteMetadata bool              `json:"delete-metadata"`
 }
 
+// ReplaceBrickReq represents replace brick request
+type ReplaceBrickReq struct {
+	SrcPeerID    string          `json:"srcpeerid"`
+	SrcBrickPath string          `json:"srcbrickpath"`
+	NewPeerID    string          `json:"newpeerid,omitempty"`
```

Comment:
New peer ID is not required, `LimitPeers` can be used if peer need to be specified